abstract_specification.md

```diff
@@ -298,11 +301,11 @@ The following naming and indexing constraints are defined for the `SOMAExperimen
 
 The SOMA API includes functional capabilities built around the [SOMA data model](#datamodel.md). The specifics of how these operations manifest in any given language and computing environment is defined elsewhere (**to be created**). Each implementation will minimally support the functional operations defined here. For example, it is likely that a Python implementation will prefer `__getattr__` over an explicit `get()` method, and will augment these functional operations with other Pythonic functionality for ease of use.
 
-In several cases an explicit Application Binary Interface (ABI) has been specified for a function, in the form of an Arrow type or construct. The choice of Arrow as both a type system and data ABI is intended to facilitate integration with third party software in a variety of computing environments.
+In several cases an explicit Application Binary Interface (ABI) has been specified for a function, in the form of an Arrow type or construct. The choice of Arrow as both a type system and data ABI is intended to facilitate integration with third-party software in a variety of computing environments.
 
-Any given storage "engine" upon which SOMA is implemented may have additional features and capabilities, and support advanced use cases -- it is expected that SOMA implementations will expose storage engine-specific features. Where possible, these should be implemented to avoid conflict with future changes to the common SOMA API. Where possible, types and variables beginning with 'soma' or 'SOMA' should be preserved for future versions of this spec.
+Any given storage engine upon which SOMA is implemented may have additional features and capabilities, and support advanced use cases: it is expected that SOMA implementations will expose storage-engine-specific features. Where possible, these should be implemented to avoid conflict with future changes to the common SOMA API. Where possible, types and variables beginning with 'soma' or 'SOMA' should be preserved for future versions of this spec.
```

Comment:
storage-engine–specific, with an en dash between "engine" and "specific"

Reply:
no thank yoiu

Reply:
No, it’s an en dash, used (among other things) as a separator to assist in grouping in compound words—this is an em dash.

Reply:
cf. [attributive compounds](https://en.wikipedia.org/wiki/Dash#Attributive_compounds); en dashes are commonly found in text and have no reason not to be in text, particularly text destined for a browser.

Reply:
Thanks for the link @thetorpedodog as well as the vigorous discussion -- but I have to draw a line at putting non-ASCII characters into markdown.

Reply:
> i think you mean em dash not en dash

I was mistaken -- en is indeed narrow than em but hyphen is narrow than both -- hyphen, en dash, em dash

These are important in typesetting but (I continue to maintain) are not appropriate in ASCII text such as markdown, wherein we have `-` and `--` (two, not three) at our disposal.

Apologies for any confusion!